### PR TITLE
Updates versions for release v6.0.0

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** 

RELEASE PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/register": "^7.12.0",
     "@babel/runtime": "^7.12.0",
-    "@cumulus/api": "7.2.1-alpha.0",
+    "@cumulus/api": "8.1.0",
     "@cumulus/aws-client": "7.0.0",
```
Comment (jennyhliu) [inline package.json:72]: We should update this as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v6.0.0] - 2021-05-03
```
Comment (jennyhliu) [inline CHANGELOG.md:10]: Update the `compare` link at the bottom of this file.
https://github.com/nasa/cumulus-dashboard#5-update-changelogmd

Comment (flamingbear) [reply CHANGELOG.md:10]: Oof I did a bad job on this today

Comment (flamingbear) [reply CHANGELOG.md:10]: 9e82dbb

File: package.json (hunk 1 above)
Comment (flamingbear) [reply package.json]: Yup, I will put up a PR to note this in the README.md as well.

**Review by jennyhliu — APPROVED** [review]
😂

**Pull request merged**